packages/cli/package.json

```diff
@@ -55,6 +56,7 @@
     "metro-react-native-babel-transformer": "^0.58.0",
     "minimist": "^1.2.0",
     "mkdirp": "^0.5.1",
+    "node-fetch": "^2.6.0",
```

Comment:
we have a util for fetching that uses `node-fetch` under the hood: https://github.com/react-native-community/cli/blob/master/packages/tools/src/fetch.ts. You can import it as: 
```js
import {fetch} from '@react-native-community/cli-tools';
```

Reply:
@thymikee I think I'm going to combine my `downloader.ts` and `fetch.ts`. The way `fetch.ts` works right now will not work well with larger files that need to be streamed instead of hold in memory.

Also is there any guidance into what should go under `packages/tools` vs `packages/cli/src/tools`?

Reply:
That's fine. You can also refactor it if it makes sense. We move stuff to `packages/tools` when other packages share the same logic.

Reply:
I've created a new method in `fetch.ts` called `fetchToTemp`. Hopefully it will also address your concern about the name.